legacy/templates/lab_report.html

```diff
@@ -1,22 +1,16 @@
 {% extends 'static_base.html' %}
-{% block javascripts %}
+
+
+{% block title %}
 <style>
   @media print{
     @page {
-        size: auto !important;
-        margin-top: 2cm !important;
-        margin-bottom: 2cm !important;
-    }
-    .static-content{
-        left: 10% !important;
-        width: 90% !important;
+      size: auto;
+      margin-left: 3cm;
+      margin-right: 3cm
```

Comment:
these no longer need to be important... 

So I was putting them in the javascript block and they were then being ignore by the production js compilation step.

Putting them in the <body> isn't the best but we don't have a hook in base.html apart from for plugins.

While we want this to be true for the lab report we do not want it to be true for the peak flow graph.